browser_windows.go

```diff
@@ -1,5 +1,10 @@
 package browser
 
+import (
+	"strings"
+)
+
 func openBrowser(url string) error {
-	return runCmd("cmd", "/c", "start", url)
```

Comment:
Firefox for android and or kindle that is what I'm using amazon Kindle fire HD updated it to 7.5.1, I do have Android app for the Google play services. There is an app installed on my kindle,  Android update 2.1 it's running as a device administrator and I am unable to deactivate IT. I try the deactivate option and it just locks the screen over and over I HAVE tried everything, including deleting it from file manager, ETC and nothing. I'm going to reset the kindle, that's the only thing I did not do yet. I DON'T KNOW WHY I'm telling you all this, this is the first time I ever commented on my file details. Thank You, 
Paula l Gray  paulalgray@hotmail.com AND paulalgray@yahoo.com AND ceoinline@gmail.com.